.cirrus.yml

```diff
@@ -431,6 +431,56 @@ task:
   on_failure: *on_failure
 
 
+task:
+  name: WindowsMingw64
+  only_if: $CIRRUS_CHANGE_MESSAGE !=~ '.*\nci-os-only:.*' || $CIRRUS_CHANGE_MESSAGE =~ '.*\nci-os-only:[^\n]*windows.*'
+  windows_container:
+    image: $CONTAINER_REPO/windows_ci_mingw64:latest
+    cpu: 4
+    memory: 4G
+  env:
+    # The default working dir is in a directory msbuild complains about
+    CIRRUS_WORKING_DIR: "c:/cirrus"
+    CCACHE_DIR: /tmp/ccache_dir
+    TEMP_CONFIG: ${CIRRUS_WORKING_DIR}/src/tools/ci/pg_ci_base.conf
+ 
```

Comment:
Could we factor out some of this into a common base task "template" with the visual studio windows task?